src/index.ts

```diff
@@ -13,7 +13,7 @@ type Specifiers = (ESTree.ImportSpecifier | ESTree.ImportDefaultSpecifier | ESTr
 type TransformModuleNameFn = (externalValue: ExternalValue) => string
 
 // constants
-const ID_FILTER_REG = /\.(js|ts|vue|jsx|tsx)$/
+const ID_FILTER_REG = /\.(mjs|js|ts|vue|jsx|tsx)(\?.*|)$/
```

Comment:
I have a question here `(\?.*|)$` . why need `|`, i don't think `|` are needed.

Reply:
because the regex syntax is like this: `(groupA|groupB)` meaning it either contains `groupA` or `groupB`

For my case we want to finish with `tsx$` or `tsx?something$`
So to support both scenarios it's like I have `(?something|)`, second group means "nothing"

I tested this with [regex tester](https://regex101.com/r/wbGZFJ/1)
![CleanShot 2022-01-24 at 15 28 00](https://user-images.githubusercontent.com/1867939/150801495-66fb2fc2-b4fa-4ff2-a18c-4d644017ef8c.gif)

Reply:
I got it! Thank you for your explanation! 😘